app/controllers/peeps.rb

```diff
@@ -0,0 +1,27 @@
+class Chitter < Sinatra::Base
+  get '/' do
+    erb(:index)
+  end
+
+  get '/peeps' do
+    erb(:'peeps')
```

Comment:
organize your view files into resource-centric sub folders.  You have a `app/views/peeps/` folder, so all peep related views should be in there.  The convention for the 'top level list' is to call it index.  So this would be `erb(:'peeps/index')`